.travis.yml

```diff
@@ -12,5 +12,6 @@ cache:
 install: true
 
 script:
+  - find ./ -name "pom.xml" -type f -exec mvn formatter:validate -f {} \;
```

Comment:
I changed to use `mvn-build-all.sh` at 6eae2783576bc9cbc349efa051103723951aebe0 commit .